polyfactory/factories/sqlalchemy_factory.py

```diff
@@ -124,8 +125,13 @@ def should_column_be_set(cls, column: Any) -> bool:
     @classmethod
     def get_type_from_column(cls, column: Column) -> type:
         column_type = type(column.type)
-        if column_type in cls.get_sqlalchemy_types():
+        if column_type in (sqla_types := cls.get_sqlalchemy_types()):
```

Comment:
Not a big deal, but I think it maybe a bit easier to read if we get `sqla_types` on a different line.

```python

sqla_types = cls.get_sqlalchemy_types()
if column_types in sqla_types:
    ...
...
```